server/config/config.go

```diff
@@ -409,6 +412,27 @@ func (c *Config) Parse(arguments []string) error {
 				c.Log.Level = c.LogLevelDeprecated
 			}
 		}
+		if c.EnableLocalTSO {
+			msg := fmt.Sprintf("enable-local-tso in %s is deprecated, use [tso.enable-local-tso] instead", c.configFile)
```

Comment:
I think there is no need to change it for now, because we might want to share the same behavior between all-in-one PD and TSO microservice, and the current behavior is perfect for TSO microservice and is evetual behavior that we want after we move all the clients to TSO microservice.

Reply:
If it's possible, we can defer this change to the future to avoid compatibility issue

Reply:
OK, I know the reason. It's to make toml.Decode(Config) happy.